### PR TITLE
Creating footer

### PR DESCRIPTION
**Pull request by iKrystyan**
I this milestone, I added:
1. Contact form
2. Submit button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,6 +180,22 @@ <h3>Skills</h3>
...
+        <form action="https://formspree.io/f/mvolzwlj" method="POST" class="contact-form">
+            <input type="text" name="Name" class="area name" placeholder="Name" maxlength="30" required>
+            <br>
+            <input type="email" name="email" class="area email" placeholder="Email" required>
+            <br>
+            <textarea name="message" class="message" maxlength="500" placeholder="Write your message here" required></textarea>
+            <br>
+        </form>
+        </div>
+        <button type="submit" class="submit">Get in touch</button>
```
Comment (Meltrust) [inline index.html]: - [ ] This button is not working.  The reason is that it is outside your `<form>` tag.  

Please kindly just move the `<button type="submit" class="submit">Get in touch</button>` line to be inside your form tag. 

image

That way, your button will start working 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Christianib003, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Please from now on, submit your next reviews to the same pull request instead of creating a new one.  Cheers!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -180,6 +180,22 @@ <h3>Skills</h3>
             </ul>
         </section>
     </section>
+    <section class="last">
```
Comment (ShoiraTa) [inline index.html:183]: - [ ] [OPTIONAL] This section has a fixed height of `800px`, and therefore the purple background is being over before the form is over: 

image

I strongly suggest to delete the ` height: 800px;` property to fix this issue.

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Christianib003 ,  

Great work on making the changes requested by a previous reviewer 👏🏻 
Your project is complete! Congratulations!  🎉💯🌟 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — COMMENTED** [review]
Hi @Christianib003 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**